spikes/npgsql-vanilla/Makefile

```diff
@@ -0,0 +1,12 @@
+help:
+	@echo "Please choose one of the following targets: start-cratedb, build, run"
+	@exit 2
+        
+start-cratedb:
+	docker run -it --rm --publish=4200:4200 --publish=5432:5432 crate:4.5.1 -C=cluster.routing.allocation.disk.threshold_enabled=false
```

Comment:
Just out of curiosity, is there a particular reason the `cluster.routing.allocation.disk.threshold_enabled` parameter gets set here but isn't present in the `docker run` example given in README.rst?

Reply:
That's because I regularly run into issues on my machine outlined at https://github.com/crate/crate/issues/11458.

Reply:
I still would keep it out :/

Reply:
All right. Addressed with f0cb179c912ce.